pkg/edvprovider/couchdbedvprovider/couchdbedvprovider.go

```diff
@@ -103,6 +114,26 @@ func (c *CouchDBEDVStore) Put(document models.EncryptedDocument) error {
 	return c.coreStore.Put(document.ID, documentBytes)
 }
 
+// GetAll fetches all the documents within this store.
+func (c *CouchDBEDVStore) GetAll() ([][]byte, error) {
+	allKeyValuePairs, err := c.coreStore.GetAll()
```

Comment:
pagination?

Reply:
performance issues

Reply:
take a look https://github.com/trustbloc/fabric-peer-ext/blob/601af71b07939521335e3185371d920e09b4e322/pkg/blkstorage/cdbblkstorage/couchdoc_conv.go#L423

Reply:
Oops... mean to add a TODO here for that. Just updated the code with a TODO: https://github.com/trustbloc/edv/issues/106

Reply:
@fqutishat A different approach will be needed here, since we don't know what keys to start on or end on... this will have to be implemented in edge-core. Keep in mind that this is on a per vault basis - for the user case that edge-agent needs, there shouldn't actually be that many documents in a single user's vault.

Reply:
Unless perhaps there's some way to adapt that approach?

Reply:
that was example the important part was the limit

Reply:
Gotcha... I've added the link you provided to the follow-up issue I created